src/optimizer/query_graph.c

```diff
@@ -2195,20 +2195,14 @@ qo_analyze_term (QO_TERM * term, int term_type)
 		  lhs_indexable = true;
 		}
 	    }
-	  else if (pt_is_multi_col_term (lhs_expr))
+	  else if (pt_is_multi_col_term (lhs_expr) && lhs_expr->flag.is_paren)
 	    {
 	      /* multi column case (attr,attr,...) is indexable for RANGE, EQ operation */
```

Comment:
It is trivial.
The comment about 'EQ operation' should be removed.

Reply:
I fixed it.